querydsl-sql/src/test/java/com/querydsl/sql/SubqueriesBase.java

```diff
@@ -155,5 +172,35 @@ public void subQuerySerialization2() {
                 serializer.toString());
     }
 
+    @Test
+    public void scalarSubQueryInClause() {
+        this.query()
```

Comment:
There are no assertions here, so we're checking an `Exception` is not thrown? Improve naming, and aid readability by putting in try catch black, catching the `Exception` and calling `fail()`. Otherwise it looks like the test forgot to assert.